### PR TITLE
Use nightly to verify for maintenance release.

### PR DESCRIPTION
**Pull request by jpeacock-zenoss**
For ZEN-26258, ZEN-26260, and ZEN-26321.

### CODE REVIEW

File: component_versions.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
         "URL": "http://zenpip.zendev.org/packages/{name}-{version}.tgz",
         "name": "metricshipper",
         "type": "download",
```
Comment (tgjonestx) [inline component_versions.json]: For version=develop, type should equal jenkins

File: component_versions.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
         "URL": "http://zenpip.zendev.org/packages/central-query-{version}-zapp.tar.gz",
         "name": "query",
         "type": "download",
```
Comment (tgjonestx) [inline component_versions.json]: For version=develop, type should equal jenkins

**Comment by tgjonestx** [comment]
Please double-check #183 and close if it's superseded by this PR

**Comment by jpeacock-zenoss** [comment]
#183 doesn't include the query service.

File: component_versions.json [hunk 3]
```diff
@@ -5,10 +5,11 @@
         "version": "1.0.1"
     },
     {
-        "URL": "http://zenpip.zendev.org/packages/{name}-{version}.tgz",
         "name": "metricshipper",
-        "type": "download",
-        "version": "1.1.2"
+        "type": "jenkins",
+        "jenkins.server": "http://jenkins.zendev.org",
+        "jenkins.job": "metricshipper-develop",
```
Comment (tgjonestx) [inline component_versions.json]: I believe `jenkins` was supposed to be a dictionary, so please try something like:
```
"jenkins": {
   "server": "http://jenkins.zendev.org",
   "job": "metricshipper-develop"
},
```

**Pull request merged**